vault/logical_system.go

```diff
@@ -2522,6 +2488,58 @@ func (b *SystemBackend) handleEnableAuth(ctx context.Context, req *logical.Reque
 	return resp, nil
 }
 
+func (b *SystemBackend) validateVersion(ctx context.Context, apiConfig APIMountConfig, pluginName string, pluginType consts.PluginType) (string, *logical.Response, error) {
+	version := apiConfig.PluginVersion
```

Comment:
I think this is the only use of `apiConfig`, so it might simplify the function signature to just pass in `apiConfig.PluginVersion` directly.

Reply:
:+1: